src/utilities/generateMeta.ts

```diff
@@ -47,3 +47,32 @@ export const generateMeta = async (args: {
     title,
   }
 }
+
+export const generateMetaPost = async (args: {
+  customTitle?: string
+  center: string
+  doc: Partial<Post>
+}): Promise<Metadata> => {
+  const { customTitle, center, doc } = args || {}
```

Comment:
Why do we need `|| {}`? Aren't we guaranteed to get `args`?

Reply:
This was already existent so I missed this, but updated since we should always have args